polaris-icons/.babelrc

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Looking at the repo there is no web code. I thought this could be removed. It is used for building the component files from the icons.